fdbcli/fdbcli.actor.cpp

```diff
@@ -565,7 +565,8 @@ void initHelp() {
 	                "change the class of a process",
 	                "If no address and class are specified, lists the classes of all servers.\n\nSetting the class to "
 	                "`default' resets the process class to the class specified on the command line. The available "
-	                "classes are `unset', `storage', `transaction', `resolution', `proxy', `master', `test', `unset', "
+	                "classes are `unset', `storage', `transaction', `resolution', `commit_proxy', `grv_proxy', "
+	                "`master', `test', `unset', "
```

Comment:
`unset` is mentioned twice here.

Reply:
Done, thanks for finding this.
And it seems to have your approval again after I committed the change.